backbone/build.gradle

```diff
@@ -33,7 +33,7 @@ android {
         exclude 'META-INF/NOTICE.txt'
     }
 
-    resourcePrefix 'rsb_'
+//    resourcePrefix 'rsb_'
```

Comment:
I'm still unable to repro, seems to behave correctly for me. Although I only see these types of errors when running inspections, not during build.

If I rename a resource so it doesn't start with rsb_, I see the warning when I run inspection. Otherwise, looks like all the resources to start with rsb_

Does this issue also occur with rss_ prefix for skin?